README.md

```diff
@@ -480,6 +480,57 @@ different version number, and also marked `default-arg-value`:
 This is due to the translation of defaults into the CFG: there is a synthetic conditional that chooses either to
 initialize the variable from the argument passed at the send, or to the default value when no value is present.
 
+Finding all references works differently in package specification (__package.rb) files. Consider the following:
+
+```ruby
+class Foo < PackageSpec
+  import Bar
+```
+
+Calling "find all references" on `Bar` in this file will return only references to `Bar` in the `Foo` package. LSP tests 
+have access to `import` and `importusage` assertions that you can use to test this functionality.
+
+```ruby
+class Foo < PackageSpec
+  import Bar
+  #      ^^^ import: bar
+```
+
+```ruby
+  class Foo::Baz
+    Bar.new
+ #  ^^^ importusage: bar
+  end
+```
+
+With these annotations, the LSP test will check if "find all references" on `Bar` in `import Bar` statement returns the `Bar.new` usage.
+
+Note that an `import` assertion is dissimilar to a `def` assertion, in that it is in fact a subclass of a `usage` assertion. 
+In this case, the `def` corresponding to an `import` is the PackageSpec declaration of the imported package. Calling "find all references"
```

Comment:
So upon arriving at the `class Bar < PackageSpec` location, clicking on go-to-def for the `class Bar` takes you right there, is that correct?

IIRC, `class Bar < PackageSpec` is secretly something like `class <PkgRegistry>::Bar < PackageSpec` or something like that. How would I ever jump from an `import Bar` to the real `Bar` class?

Reply:
Don't think we have a way currently to do that. I think that stopped working after we switched to the visibility checker design.

Reply:
This PR improves the story a little, but does not solve that problem.

Reply:
But we can totally fix that in a separate change